src/platforms/salesforce/schemas.ts

```diff
@@ -20,31 +31,37 @@ export const salesforceQueryResponse = validators.object({
 // -
 // User
 // -
-export const salesforceUser = validators.object({
-  Id: z.string(),
-  FirstName: z.string(),
-  LastName: z.string(),
-  Email: z.string().email(),
-  CreatedDate: validators.date(),
-  LastModifiedDate: validators.date(),
-});
+export const salesforceUser = validators
+  .object({
+    Id: z.string(),
+    FirstName: z.string(),
+    LastName: z.string(),
+    Email: z.string().email(),
+    CreatedDate: validators.date(),
+    LastModifiedDate: validators.date(),
+  })
+  .partial()
+  .required(requiredFields);
 
 // -
 // Contact
 // -
-export const salesforceContact = validators.object({
-  Id: z.string(),
-  FirstName: z.string(),
-  LastName: z.string(),
-  Title: z.string(),
-  Email: z.string().email(),
-  Phone: z.string(),
-  MobilePhone: z.string(),
-  CreatedDate: validators.date(),
-  LastModifiedDate: validators.date(),
-  AccountId: z.string(),
-  OwnerId: z.string(),
-});
+export const salesforceContact = validators
+  .object({
+    Id: z.string(),
+    FirstName: z.string(),
+    LastName: z.string(),
+    Title: z.string(),
+    Email: z.string().email(),
```

Comment:
As an fyi, I'm not sure if Salesforce has UI validation for email fields. I think this might just be a string field - did you check? I know at least for the API this can be whatever

Reply:
They might do some checks, but actually in retrospect I think we should keep the validations/transformations here fairly limited (probably just limit it to Dates)